.github/workflows/graalvm.yml

```diff
@@ -56,6 +59,18 @@ jobs:
            GRADLE_ENTERPRISE_CACHE_USERNAME: ${{ secrets.GRADLE_ENTERPRISE_CACHE_USERNAME }}
            GRADLE_ENTERPRISE_CACHE_PASSWORD: ${{ secrets.GRADLE_ENTERPRISE_CACHE_PASSWORD }}
            PREDICTIVE_TEST_SELECTION: "${{ github.event_name == 'pull_request' && 'true' || 'false' }}"
+      - name: Add build scan URL as PR comment
+        uses: actions/github-script@v5
+        if: github.event_name == 'pull_request' && failure()
```

Comment:
Why limiting this to failures?

Reply:
This is how it’s in the Gradle docs. Would you like to have it all the time?

Reply:
I think so, yes, there's no harm in always having the link.